Submissions/Shah_Week2.playground/Contents.swift

```diff
@@ -0,0 +1,64 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let floatValue: Float = 4
+
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width)
+
+/*Binary operator '+' cannot be applied to operands of type 'String' and 'Int' */
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+let miles:Float = 3000
+let name = "Ray"
+let carmiles = "\(name)'s car has travelled \(miles + 0.5) miles in her last trip"
+
+//: Control Flow
+//: 1. Change optional Name to nil. What greeting do you get? Add an else clause that sets a different greeting if optionalName is nil.
```

Comment:
unanswered question: What greeting do you get if you change `optionalName` to nil?